build/build-sdk-images/node/publish.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+AGONES_PATH=$1
+RELEASE_VERSION=$2
+
+npm login --registry=https://npm.pkg.github.com
+
+cd $AGONES_PATH/sdks/nodejs
+
+npm version $RELEASE_VERSION
```

Comment:
How/when does this script get run?

I think updating the version number should be a careful change and follow semver, so potentially even a manual step. Or are these variables automatically populated and the aim is to keep the package version the same as the release version? If that is the case then what will happen when there are no SDK changes and how do we still comply with semver?

Reply:
OK I've read #679 . Perhaps best to first focus on the publishing part and ensure it works and can look at again if we want to do version sync?

Reply:
I'm happy to remove the version update. i was going to handle the release part - sounds like a check in the release checklist that we update the version (we can do this by hand?) would work.

So for this PR, we could drop the `npm version $RELEASE_VERSION` command?

Reply:
Correct. I only added that as a potential way to address an earlier comment from you @markmandel.

@steven-supersolid I believe the idea is to run this from `release.mk` but I was going to leave that part to Mark.

Reply:
Looks good and it seems we keep the SDK version the same as the Agones version?

Reply:
That was the intent... I took out the `npm version` command for now though, we can always add it later if we want.